drodrpg/Texts/MIDs.h

```diff
@@ -1556,7 +1557,7 @@ enum MID_CONSTANT {
   MID_Snow = 1151,
   MID_LightCrossfade = 1278,
 
-  MID_END_UNUSED=1748
+  MID_END_UNUSED
```

Comment:
This change doesn't seem correct.  This end MID sentinel should be explicitly set as one above the max MID value in the set.  This should be done automatically when running DRODUtil following these sample steps (I keep these in a .bat file to run whenever I add a new text):

```
#Sample build steps for generating a fresh barebones drod5_0.dat file
cd C:\Caravel\DROD_Dev\DRODUtil\Release
drodutil delete
drodutil create
drodutil import C:\Caravel\DROD_Dev\Texts
```

Reply:
I've fixed this up manually, and I'll keep an eye out in case this happens again in the future.